server/package.json

```diff
@@ -1,28 +1,14 @@
 {
-  "name": "@report-bot/server",
+  "name": "server",
   "version": "1.0.0",
-  "description": "Slack bot for scraping the content of messages and storing them in a database.",
-  "main": "src/index.js",
-  "repository": "git@github.com:vacuumlabs/report-bot.git",
-  "author": "Tomas Fabry <tomas.fabry@vacuumlabs.com>",
   "license": "Unlicense",
   "scripts": {
-    "clean": "rimraf dist && rimraf node_modules",
+    "build": "rm -rf dist && babel src -d dist",
+    "clean": "rm -rf dist && rm -rf node_modules",
     "dev": "nodemon --exec babel-node --inspect src/index.js",
```

Comment:
I made a clean install and I cannot run this command. Can you please fix this?

Reply:
Missing `@babel/node` (same as `rimraf`). I added it.